src/main/java/com/car/app/tests/CarServiceTest.java

```diff
@@ -0,0 +1,95 @@
+package com.car.app.tests;
```

Comment:
Please move the test classes to the folder: src/test/java
This is the default place for unit test: https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html